vpr/src/pack/cluster.cpp

```diff
@@ -1390,6 +1536,18 @@ static enum e_block_pack_status try_pack_molecule(t_cluster_placement_stats* clu
     return block_pack_status;
 }
 
+static void record_molecule_failure(t_pack_molecule* molecule, t_pb* pb) {
+    //Record the failure of the molecule in this cluster in the current pb stats
```

Comment:
would this work if I only record for the first atom? if it is fine, only check in one spot, and write that that is the convention